internal/utilities_linux.go

```diff
@@ -0,0 +1,20 @@
+package internal
```

Comment:
To continue using this code on darwin (macOS) and BSD, this file should not be named `_linux.go`. Instead use a [build constraint](https://golang.org/pkg/go/build/) comment:

```
// +build !windows
```